### PR TITLE
[TIMOB-16218] Fix Ti.UI.RefreshControl.beginRefreshing, refactor proxy and docs

### PR DESCRIPTION
**Pull request by hansemannn**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-16218

### CODE REVIEW

File: apidoc/Titanium/UI/RefreshControl.yml [hunk 1]
```diff
@@ -37,7 +41,14 @@ properties:
...
 events:
   - name: refreshstart
-    summary: Fired in response to a user initiated action to refresh the contents of the table view or list view.
+    summary: Fired in response to a user initiated an action to refresh the contents of the table view, list view or scroll view.
+    properties:
+      - name: bubbles
+        summary: This is false. This event does not bubble
+        type: Boolean
+  - name: refreshend
+    summary: Fired in response to a user finished an action to refresh the contents of the table view, list view or scroll view.
```
Comment (AngelkPetkov) [inline apidoc/Titanium/UI/RefreshControl.yml]: This sentence doesn't make sense,reword please.

**Comment by AngelkPetkov** [comment]
FT and CR approved, address the comment above please and will merge 👍

File: apidoc/Titanium/UI/RefreshControl.yml (hunk 1 above)
Comment (hansemannn) [inline apidoc/Titanium/UI/RefreshControl.yml]: It is the native description Apple provides and is copied & adjusted from `refreshstart`.

**Comment by AngelkPetkov** [comment]
Merging!

**Pull request merged**